TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Title
+
+These documents have names that are short noun phrases. For example, "ADR 1: Deployment on Ruby on Rails 3.0.10" or "ADR 9: LDAP for Multitenant Integration"
+
+## Context
+
+This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+## Decision
+
+This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+## Status
+A decision may be "proposed" if the project stakeholders haven't agreed with it yet, or "accepted" once it is agreed. If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement.
+
+## Consequences
+
+This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+The whole document should be one or two pages long. We will write each ADR as if it is a conversation with a future developer. This requires good writing style, with full sentences organized into paragraphs. Bullets are acceptable only for visual style, not as an excuse for writing sentence fragments. (Bullets kill people, even PowerPoint bullets.)
```

Comment:
> (Bullets kill people, even PowerPoint bullets.)

I feel like this joke has the ability to be taken the wrong way and a template can be used by people without context. I'd suggest removing it.

Reply:
Understood, it was copy pasted from another template without much thought. So happy to change.